CHANGELOG.md

```diff
@@ -55,8 +55,8 @@ ZK Settings: zksolc version [1.5.3](https://github.com/matter-labs/era-compiler-
 
 | Contract Name | EVM                                          | ZKSync                                       |
 | ------------- | -------------------------------------------- | -------------------------------------------- |
-| `Safe`        | `0x85537a85f40c8B23e03bf95087f00b6bA1c11b3D` | `0xb55299340729B4F47ac2f00c955E7D7592fA76FC` |
-| `SafeL2`      | `0x0859B89940E228513a8774079cB098c9ab40937D` | `0x1373d363aE75d6854e646cdFc475833010189f9a` |
+| `Safe` | `0x85537a85f40c8B23e03bf95087f00b6bA1c11b3D` | `0xb55299340729B4F47ac2f00c955E7D7592fA76FC` |
+| `SafeL2` | `0x0859B89940E228513a8774079cB098c9ab40937D` | `0x1373d363aE75d6854e646cdFc475833010189f9a` |
```

Comment:
I actually liked the formatting where the columns were aligned - just curious about why they changed.

Reply:
I think that was prettier or so, not done manually.